src/controller.py

```diff
@@ -119,10 +119,14 @@ def run_command_and_get_its_response(self) -> dict:
             emulator.wipe_device()
             return {"response": "Device wiped"}
         elif self.command == "emulator-apply-settings":
-            emulator.apply_settings(
-                self.request_dict["passphrase_always_on_device"],
-            )
-            return {"response": f"Applied setting on emulator {self.request_dict}"}
+            # Relaying all the relevant fields from the request, to make sure
+            #   the client is notified when it sends an unknown field
+            #   and the function will throw an Exception
```

Comment:
Not very important but we do not add the two spaces in the comments for next lines usually.

Reply:
Sure, I will "fix" the comments, but I cannot promise I will not do it again in the future, as I do it automatically - I think it really quickens the text comprehension, one can immediately see that this "block" is one sentence (or logic block)

Reply:
Feel free to keep it. I really do not have much of an opinion on that and you are right we should either enforce it in CI or forget about it :)).